JobConfig/cosmic/genconfig_cosmic_general.txt

```diff
@@ -0,0 +1,27 @@
+//
+// Configuration for CosmicDYB
+// (general simulation)
+//
+// Ralf Ehrlich, 2015
+
+bool conversionGun.do = false;
+bool cosmicDYB.do = true;
+bool picapture.do = false;
+bool ejectedProtonGun.do = false;
+bool decayinorbitGun.do = false;
+bool piEplusNuGun.do = false;
+
+double cosmicDYB.mean =  -1.1; 
+double cosmicDYB.muEMin = 0.5e3;
+double cosmicDYB.muEMax = 5000e3;
+double cosmicDYB.dx     = 10000;
+double cosmicDYB.dy     = 0;
+double cosmicDYB.dz     = 20000;
+double cosmicDYB.direction = "Negative_y";
+vector<double> cosmicDYB.productionCenterInMu2e = {-3904,0,10200};
```

Comment:
Is this not the Mu2e detector coordinate center?  Could it instead be read from the GeometryService?  Fixing this might be outside this PR.

Reply:
This is for the Daya Bay cosmic ray generator, which can be configured with 2D production planes or a 3D "production box". The center can be placed anywhere, and we have used many different configurations over the years. So reading one particular set of coordinates from the GeometryService may only be useful, if we used it as default parameter which can be overwritten in the config file.